enos/modules/run_workloads/jobs/docker-system.nomad.hcl

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+variable "alloc_count" {
+  type    = number
+  default = 1
+}
+
+job "system-docker" {
+  type = "system"
+
+  group "system-docker" {
+
+    task "system-docker" {
+      driver = "docker"
+
+      config {
+        image   = "alpine:latest"
+        command = "sh"
+        args    = ["-c", "while true; do sleep 30000; done"]
```

Comment:
For service/system jobs, we should probably use a workload where we can assert more about it than "is the alloc running?". `busybox httpd` would let us run a network service, so that we're exercising things like restoring CNI. Ok to leave for this PR, but let's come back to this too.